### PR TITLE
Adding Farsi/Persian language.

### PR DESCRIPTION
**Pull request by irajtaghlidi**
Add Farsi language, related to #35

### CODE REVIEW

File: app/config/locale/translations/fa.countries.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    'BR' => 'برزیل',
+    'BB' => 'باربادوس',
+    'BN' => 'برونئی',
+    'BT' => 'بوتان',
+    'BW' => 'بوتسوانا',
+    'CF' => 'جمهوری آفریقای مرکزی',
+    'CA' => 'کانادا',
+    'CH' => 'سوئیس',
+    'CL' => 'شیلی',
+    'CN' => ']dk',
```
Comment (spielers) [inline app/config/locale/translations/fa.countries.php:36]: Hey @irajtaghlidi please retranslate this

**Review by spielers — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/fa.countries.php (hunk 1 above)
Comment (irajtaghlidi) [reply app/config/locale/translations/fa.countries.php]: Thank you @spielers for your review, I fixed the 'CN' typo one.

**Review by spielers — APPROVED** [review]
Hey, @irajtaghlidi  Thanks for your contributions will get you merged!

**Review by spielers — COMMENTED** [review]
Hey, @irajtaghlidi thanks!

**Comment by irajtaghlidi** [comment]
it's my pleasure @spielers.

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**